lib/index.js

```diff
@@ -32,24 +33,34 @@ function getStatsdApi ({ hostname, port, lynxOptions, transmit, prefix }) {
   }
 
   const lynx = new Lynx(hostname, port, lynxOptions)
+  const sdc = new SDC({ host: hostname, port: port, tcp: true })
```

Comment:
if `statsd-client` can transmit via `tcp` _and_ `udp`, I think it'd be best to swap out `lynx` entirely for `statsd-client`.

Reply:
Your suggestion has been added